app/Http/Controllers/BlogController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Blog;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class BlogController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+        $blogs = Blog::all();
+        return view('blog/index', ['blogs'=>$blogs]);
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        //
```

Comment:
why have this comment without content ? if not comment please remove

Reply:
i fixed